helm/rollup/files/genesis/geth-genesis.json

```diff
@@ -17,6 +17,10 @@
     "difficulty": "10000000",
     "gasLimit": "8000000",
     "alloc": {
-        "{{ .Values.genesisAllocAddress }}": { "balance": "300000000000000000000" }
+    {{- $accounts_length := len .Values.config.rollup.genesisAccounts  }}
```

Comment:
You can use some template helpers to do this so that you don't have any logic for the comma.
e.g.
`"accounts": {{ toPrettyJson .Values.genesisAccounts | indent 4 | trim }},`

https://github.com/astriaorg/dev-cluster/pull/95/files#diff-29a69d8e99ecd4c315c4d8388575938d212f3706578b67ce7c3e6c6089a81c43R4

Reply:
Part of my goal here was to mimic the shape of data between the two values, and if I just do the pretty json I can't have the shapes match. Viewing the values format as product and wanting consistency around them